faust/types/app.py

```diff
@@ -189,6 +189,13 @@ def task(self, fun: TaskArg) -> Callable:
     def timer(self, interval: Seconds, on_leader: bool = False) -> Callable:
         ...
 
+    @abc.abstractmethod
+    def crontab(self,
+                cron_format: str,
+                tz: any = None,
```

Comment:
@ask `tz` needs to be a `tzinfo` type that I was planning to get from `pytz`, but `pytz` isn't included in the main requirements and didn't want to add one more requirement to get a type, thoughts?

Reply:
In Python 3.6 you can use `datetime.timezone`, this is what I've used for typing timezones so far.

Looking at mypy and their type header stubs though, I see that pytz timezones are typed with the abstract base class `datetime.tzinfo`

In datetime the `timezone` class is defined as:

```
class timezone(tzinfo):
```

so to support pytz we should change all `timezone` types to `datetime.tzinfo`

Reply:
@ask I'll look into `datetime.timezone` and `datetime.tzinfo` and then I'll change the types, thanks.